plugins/scale/scale-title-filter.cpp

```diff
@@ -134,36 +206,25 @@ class scale_title_filter : public wf::plugin_interface_t
         auto xkb_state = keyboard->xkb_state;
         xkb_keycode_t keycode = raw_keycode + 8;
         xkb_keysym_t keysym   = xkb_state_key_get_one_sym(xkb_state, keycode);
+        auto& filter = share_filter ? get_instance() : local_filter;
```

Comment:
Maybe we can extract this bit in its own short function? Seems it is repeated quite a lot.